eslint.config.mjs

```diff
@@ -31,6 +32,16 @@ export default [
   ...eslintPluginVue.configs['flat/vue2-recommended'],
   ...vuejsAccessibility.configs["flat/recommended"],
   ...intlifyVueI18N.configs['flat/recommended'],
+  {
+    plugins: {
+      jsdoc
+    },
+    rules: {
+      'jsdoc/check-param-names': 'error',
+      'jsdoc/no-multi-asterisks': 'error',
+      'jsdoc/check-alignment': 'error'
+    }
+  },
```

Comment:
I haven't tested this yet but placing it in it's own group without any constraint on which file extensions to apply it to, from my understanding would mean that these eslint rules are run on the JSON and YAML files too. We should probably add it directly to the relevant existing config objects or limit this to the relevant files.